activities/management/commands/jslps_people_latest.py

```diff
@@ -0,0 +1,140 @@
+import urllib2
+import unicodecsv as csv
+import xml.etree.ElementTree as ET
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from geographies.models import *
+from people.models import *
+from programs.models import *
+import jslps_data_integration as jslps
+
+class Command(BaseCommand):
+	def handle(self, *args, **options):
+		#read xml from url
+		url = urllib2.urlopen('http://webservicesri.swalekha.in/Service.asmx/GetExportSRIRegistrationDataNew?pUsername=admin&pPassword=JSLPSSRI')
+		contents = url.read()
+		xml_file = open("jslps_data_integration_files/person.xml", 'w')
+		xml_file.write(contents)
+		xml_file.close()
+
+		partner = Partner.objects.get(id = 24)
+		user_obj = User.objects.get(username="jslps_bot")
+		csv_file = open('jslps_data_integration_files/person_error.csv', 'wb')
+		wtr = csv.writer(csv_file, quoting=csv.QUOTE_ALL)
+		tree = ET.parse('jslps_data_integration_files/person.xml')
+		root = tree.getroot()
+		print len(root.findall('SRIRegistrationData'))
+		for c in root.findall('SRIRegistrationData'):
+			pc = c.find('MemID').text
+			pn = unicode(c.find('MemberName').text)
+			if c.find('FatherName') is not None:
+				pfn = unicode(c.find('FatherName').text)
+			else:
+				pfn = ''
+			if c.find('PhoneNo') is not None:
+				phone = c.find('PhoneNo').text
+			else:
+				phone = ''
+			if c.find('Age') is not None:
+				age = int(c.find('Age').text)
+			else:
+				age = None
+			if c.find('Gender') is not None:
+				gender = c.find('Gender').text
+			else:
+				gender = 'F'
+			vc = c.find('VillageCode').text
+			gc = c.find('GroupCode').text
+
+			try:
+				village = JSLPS_Village.objects.get(village_code = vc)
+			except JSLPS_Village.DoesNotExist as e:
+				wtr.writerow(['JSLPS village not EXIST: '+str(pc), vc, e])
+				continue
+			error = 0
+			try:
+				group = JSLPS_Persongroup.objects.get(group_code = gc)
+			except JSLPS_Persongroup.DoesNotExist as e:
+				wtr.writerow(['JSLPS group not EXIST: '+str(pc), gc, e])
+				error = 1
+
+			if error == 1:
```

Comment:
In both the cases, you are doing the same thing.

Reply:
corrected